src/components/create-category-component/CreateCategory.tsx

```diff
@@ -0,0 +1,112 @@
+import React from 'react'
+import { TextField, Button} from '@material-ui/core'
+import { string } from 'prop-types'
+// import { Redirect } from 'react-router-dom'
+
+
+
+// function checkCategory(input: { value: string; }) {
+
+//     // Check if input contains a digit
+//     if (/\d/.test(input.value)) {
+//         alert('Name contains a number');
+
+//     }
+
+//     // Check if input is empty
+//     if (input.value === '') {
+//         alert('Name is empty');
+//     }
+
+//     return true;
+// }
```

Comment:
No commented out code should exist, I see you left for later, but you shouldn't make a pr until its actually done